### PR TITLE
Vendor pliny as a gem in vendor, add gemspec/etc

### PR DESCRIPTION
**Pull request by pedro**

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -17,5 +17,3 @@ class MiniTest::Spec
 end
 
 ENV.update(Pliny::Utils.parse_env("#{Pliny.root}/.env.test"))
-
-Pliny.initialize!
```
Comment (brandur) [inline test/test_helper.rb:21]: BTW, I think you need to leave this one and flip the other one I think. The idea is to initialize after we have enough info to do so (i.e. after we've pulled critical config in from `.env.test` on the line above).

File: vendor/pliny/pliny.gemspec [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Gem::Specification.new do |gem|
+  gem.name    = "pliny"
+  gem.version = "0.0.1"
+
+  gem.author      = "Brandur and Pedro"
```
Comment (brandur) [inline vendor/pliny/pliny.gemspec:5]: I think both `author` and `email` can be arrays here :)

File: test/test_helper.rb (hunk 1 above)
Comment (pedro) [inline test/test_helper.rb:21]: oh +1! fixing

**Comment by brandur** [comment]
Rocking!! Left a couple small comments, but this is definitely the Right Way Forward.

**Comment by pedro** [comment]
Thanks!

**Pull request merged**